.cargo/audit.toml

```diff
@@ -1,5 +1,4 @@
 [advisories]
 ignore = [
-    "RUSTSEC-2020-0071", # `time` localtime_r segfault
     "RUSTSEC-2022-0093", # `ed25519-dalek` Oracle attack, remove once https://github.com/deislabs/bindle/pull/368 is merged
```

Comment:
Is this one still needed? The linked bindle PR has been merged

Reply:
there has been no release